Reflection/ReflectionEnum.php

```diff
@@ -33,5 +35,6 @@ public function isBacked(): bool {}
     /**
      * @return ReflectionType|null
      */
+    #[LanguageLevelTypeAware(['8.2' => 'null|ReflectionNamedType'], default: 'null|ReflectionType')]
     public function getBackingType(): ?ReflectionType {}
```

Comment:
@DmitryTronin Please remove ?ReflectionType from signature as far as you declare type in attribute